docs/changelog/94620.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 94620
```

Comment:
I don't think you need a changelog entry for a docs change, so you should be able to delete this file.